pkg/controller/scansettingbinding/scansettingbinding_controller.go

```diff
@@ -211,6 +214,52 @@ func (r *ReconcileScanSettingBinding) Reconcile(ctx context.Context, request rec
 		}
 	}
 
+	scanSetting := &compliancev1alpha1.ScanSetting{}
+	if instance.SettingsRef != nil {
+		ssb := instance.DeepCopy()
+		err = r.Client.Get(context.TODO(), types.NamespacedName{Name: instance.SettingsRef.Name, Namespace: request.Namespace}, scanSetting)
+		if err != nil {
+			return reconcile.Result{}, fmt.Errorf("Failed to get ScanSetting: %s", err)
+		}
+		if scanSetting.ComplianceSuiteSettings.Suspend {
```

Comment:
it might be a nitpicking here, So I wonder if should we set SSB conditions based on whether the current status of the CR matches the desired suspended status? so we don't set the condition status on every reconcile loop.

Reply:
similar to `scanSettingBindingStatusNeedsUpdate`

Reply:
Fixed - good call.